Graylog2/graylog2-server
# Move lock file check for data dir to later step

## Description
To reduce the time needed for changing e.g. permission to the data dir when migrating from OpenSearch to the data node, the lock file check is moved to before startup of the data node's configured OS.
Also, some more information is given to the user in the last steps of the migration wizard.

/nocl

## Motivation and Context
fixes #18630 

## How Has This Been Tested?

- Make data node directory non-writable for user running data node
- start up data node
- walk through migration
- check error in last step of migration
- make directory writable
- restart data node
- finish migration

## Types of changes


## Checklist:
